composer.json

```diff
@@ -34,10 +34,5 @@
     },
     "autoload-dev": {
         "psr-4": { "SendinBlue\\Client\\" : "test/" }
-    },
```

Comment:
Hi @bastiendonjon 

Can you please retain this part?
It is needed

Reply:
yes